google-datacatalog-kafka-connector/src/google/datacatalog_connectors/kafka/prepare/datacatalog_tag_template_factory.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud import datacatalog_v1beta1
+
+from google.datacatalog_connectors.kafka.config import TagTemplateConstants
+
+
+class DataCatalogTagTemplateFactory:
```

Comment:
Is this Kafka-specific at all? If not perhapes should be moved to some common package?

Reply:
I think this could be part of a commons `message broker` package, like we have for `rdbms`. Since it has cluster and topic templates logic that could be reused.

I'd suggest moving this and some other classes once we have another message broker connector.